events/0916_seo.html

```diff
@@ -0,0 +1,178 @@
+                    <li><strong>Detroit Craftsman Guild - More Functional Programming: Meet Haskell! with Jason Stolaruk</strong><br /><small>Tuesday 9/20 @ Deroit Labs</small></li>
+                    <li><strong class="blue">Code &amp; Coffee</strong><br /><small>Wednesday 9/21 @ Espresso Royale</small></li>
+                    <li><strong>A2 Cofee House Coders - Meetup</strong><br /><small>Wednesday 9/21 @ Espresso Royale</small></li>
+                    <li><strong class="blue">Intro to Version Control: Git, Github, &amp; The Terminal Workshop</strong><br /><small>Sat 10/8 &amp; Sun 10/9 @ Arbor Networks</small></li>
+
+<!--                     <li><strong class="red">GDI Toledo - Code and Coffee</strong><br /><small>Saturday 8/27 @ Starbucks - Holland, OH</small></li>
+ -->                    
+                </ul>
+                <p class="clear box--small green"><em>Plus soooo many more...</em></p>
+                <p class="clear copy--small"><a href="http://bit.ly/gdiaa-community">bit.ly/gdiaa-community</a></p>
+                <aside class="notes"></aside>
+            </section>
+
+            <!-- Sponsors -->
+            <section>
+                <h3>Sponsors!</h3>
+                <img src="images/sponsors-3.2016.png" class="img--bare box--small" />
+            </section>
+
+            <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/Nexient.png" class="img--bare box--small" height="500" />
+            </section>
+
+<!--             <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/TIY-Logo-1000px.png" class="img--bare box--small" height="450" />
+                <div class="box--small" style="margin-bottom:0"><a target="_blank" href="http://bit.ly/2bx5flh">Sign In</a></div>
+            </section>
+ -->
+            <!-- Introduce Organizer -->
+            <section>
+                <div class="box--small">
+                    <h3 class="event__title">Thanks Karen!!</h3>
+                    <img src="images/karen.jpg" class="img--bare box" width="254" />
+                </div>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Workshop Slides</h3>
+                <div class="box"><a target="_blank" href="http://www.gdiannarbor.com/events/intro-seo/">gdiannarbor.com/events/intro-seo</a></div>
+            </section>
+
+            <!-- Link to Slack -->
+            <section>
+                <img src="images/slack.png" class="img--bare box" />
+                <p>Join us on Slack - <a target="_blank" href="http://bit.ly/gdiaa-slack">bit.ly/gdiaa-slack</a></p>
+                <br />
+<!--                 <p>Already on slack, join the group - <a target="_blank" href="https://gdiaa.slack.com/messages/0824-introcoding/">0824-introcoding</a></p>
+ -->            </section>
```

Comment:
Let's just remove this whole slide - I don't want slack to be a distraction at the beginnging of class if we're not going to use it. So we'll add a slide like this to the end of the actual workshop slides instead.